src/tests/testSetup.ts

```diff
@@ -0,0 +1,60 @@
+import { runMutations } from "automutate";
+import fs from "node:fs";
```

Comment:
[Refactor] Nit: I'm a fan of switching to the async APIs whenever possible. Now that this is within Vitest and supports them, IMO it'd be nice to switch over. Do you have a strong preference here?

```suggestion
import fs from "node:fs/promises";
```

(this is not a big deal to me and I wouldn't block on it if you do care one way or another)

Reply:
Makes sense, let's leave as-is. Good points!